src/modules/ekf2/ekf2_main.cpp

```diff
@@ -1390,6 +1393,16 @@ void Ekf2::run()
 					lpos.dist_bottom = _rng_gnd_clearance.get();
 				}
 
+				// update ground effect flag based on terrain estimation
+				if (lpos.dist_bottom_valid && lpos.dist_bottom < _gnd_effect_deadzone.get()) {
```

Comment:
@EliaTarasov I suggested to use _gnd_effect_deadzone.get() here but I don't think that makes sense. Should we use a fixed value like 1m or something?

Reply:
@RomanBapst yes we may but i think it's better to add another parameter for this purpose. So `_gnd_effect_deadzone` would be both for flag and for barometer innovations,
and say `_gnd_effect_distance` would be for distance above ground where we may expect such innovations.
What do you think?

Reply:
@RomanBapst any thoughts?

Reply:
@EliaTarasov Apologies for the delay! Yes, that sounds like a good idea. Let's do those changes and then get this PR in. After this I want to continue taking into account takeoff as well (for the case where we don't have a terrain estimate). I already talked to @priseborough and he is ok with removing the local position dependency in the estimator. I think it's anyway a good idea to not put this kind of logic into the estimator itself but have it reside here.

Reply:
@RomanBapst Great, thanks! I've added the `distance` parameter.

Reply:
@RomanBapst Good to go?